extensions/amp-story-education/0.1/amp-story-education.js

```diff
@@ -51,6 +52,11 @@ const buildNavigationEl = element => {
   `;
 };
 
+/** @enum {string} */
+const Screen = {
+  ONBOARDING_NAVIGATION: 'on',
```

Comment:
nit: maybe be more descriptive about what this screen is?  These names (the minified `on` version) are permanent since they'll be embedded in other viewers that we can't change, but I wonder if e.g. we experimented with multiple versions of onboarding, how we would tell them apart

Reply:
Do you mean descriptive as in adding a comment in our codebase, or changing the `on` word to something more verbose?

If the latter, wdyt of something like: `ONBOARDING_NAVIGATION_TAP_AND_SWIPE: 'ontas`?

Reply:
I meant the latter, and yep, your suggestion LGTM!